examples/use_completions.rs

```diff
@@ -60,11 +61,53 @@ async fn main() {
     let mistral_completion = Completions::new(model, &mistral_api_key, None, None);
 
     match mistral_completion
-        .debug()
         .get_answer::<TranslationResponse>(instructions)
         .await
     {
         Ok(response) => println!("Mistral response: {:?}", response),
         Err(e) => eprintln!("Error: {:?}", e),
     }
+
+    // Get answer using Google GeminiPro
+
+    /*
+    // ********** Code to obtain auth token for Vertex AI API using a GCP service account and key
+    let model = GoogleModels::GeminiProVertex;
+
+    // To authenticate Google we need to use a key associated with a GCP service account with correct grants
+    // Load your service account key from a file or an environment variable
+    // Remember to turn on the yup_oauth2 imports
+    let service_account_key = read_service_account_key("secrets/gcp_sa_key.json")
+        .await
+        .unwrap();
+
+    // Authenticate with your service account
+    let auth = ServiceAccountAuthenticator::builder(service_account_key)
+        .build()
+        .await
+        .unwrap();
+    let google_token = auth
+        .token(&["https://www.googleapis.com/auth/cloud-platform"])
+        .await
+        .unwrap();
+    let google_token_str = &google_token.token().unwrap();
+    // ********** End Google Vertex AI authentication code
+    */
```

Comment:
Your thoughts on this? To use Vertex AI user need to have a GCP account with a service account. This walks them through how to leverage that. 

But to use Google AI Studio they only need workspace account and an API key. I have an example for that bellow too.

Is this too much hand holding?

Reply:
Sounds good. I was planning to do a follow-up PR to clean up documentation.